src/sections/AddressInfoSection.tsx

```diff
@@ -196,6 +185,7 @@ interface AddressTransactionRowProps {
 }
 
 const AddressTransactionRow: FC<AddressTransactionRowProps> = ({ transaction, addressId }) => {
+  const theme = useTheme()
   const t = transaction
```

Comment:
I know it's not part of this PR, but this could be simplified by adding in directly in the component's props:
```
({ transaction: t, addressId })
```

Reply:
Done